testsuite/features/support/commonlib.rb

```diff
@@ -617,7 +617,7 @@ def channel_is_synced(channel)
 # @return [ApiTestXmlrpc, ApiTestHttp] The created API client.
 def new_api_client
   ssl_verify = !$is_gh_validation
-  if product == 'SUSE Manager'
+  if $debug_mode || product == 'SUSE Manager'
```

Comment:
With this added again, we will not be able to use the HTTP API with debug mode, which means no debug mode on Uyuni.

Reply:
Sorry, a leftover, without it, I can't runin debug mode on my IDE in MacOS due to certificate issue with SSL.
Personally, I would like to keep as it is, but ok, I will remove from this PR, it was a leftover.

Reply:
Sure, fine with me for now :+1: